optuna_dashboard/_app.py

```diff
@@ -411,6 +412,45 @@ def save_study_note(study_id: int) -> BottleViewReturn:
         response.status = 204  # No content
         return {}
 
+    @app.post("/api/studies/<trial_id:int>/tell")
```

Comment:
How about renaming `studies` with `trials`?

```suggestion
    @app.post("/api/trials/<trial_id:int>/tell")
```